schemas/core/components/terms.json

```diff
@@ -99,6 +121,64 @@
           "type": ["string", "number"]
         }
       }
+    },
+    "cancellation": {
+      "type": "object",
+      "properties": {
+        "cancellable": {
+          "type": "boolean",
+          "description": "A flag indicating whether this booking can be cancelled in 'RESERVED' state"
+        },
+        "cost": {
```

Comment:
You could reference this from `/components/cost.json`
There's a hiccup here. We don't want to have here cost, but fares if possible

Reply:
In term wise, I believe using `cost` is appropriate naming as this is not fare for travel but cost of changing or cancelling your ticket (one could argue that `price` is more appropriate...). What it comes to supported currencies, this cost is coming from TSP, which means that supporting `WMP` is not possible.

Reply:
then this should accept either cost or fare. It should be converted after response coming out of TSP. Client would then be able to display it properly and also backend does not need to convert it every time